hdk-rust/tests/integration_test.rs

```diff
@@ -57,10 +57,25 @@ fn start_holochain_instance() -> (Holochain, Arc<Mutex<TestLogger>>) {
     ]);
     let mut dna = create_test_dna_with_cap("test_zome", "test_cap", &capabability, &wasm);
 
-    dna.zomes.get_mut("test_zome").unwrap().entry_types.insert(
-        String::from("validation_package_tester"),
-        EntryTypeDef::new(),
-    );
+    // TODO: construct test DNA using the auto-generated JSON feature
```

Comment:
Can you please make sure this gets documented in a ticket or a leaf on the tree?

Reply:
https://realtimeboard.com/app/board/o9J_kyiXmFs=/?moveToWidget=3074457346314650824